conda-env-mod

```diff
@@ -297,15 +296,16 @@ function create_module()
 	echo >> "${MODULE_FILE}"
 
 	if [[ $Opt_LOCAL_PY -eq 0 ]]; then
+		# if user put down base-python, then use Python from base Anaconda
 		cat <<-EOF >> "${MODULE_FILE}"
 			-- This line is deliberately commented out.
 			-- We want to use Python from base Anaconda.
 			-- prepend_path("PATH",modroot.."/bin")
 		EOF
 	else
+		# python from the environment (python -V) is added to PYTHONPATH by default
 		cat <<-EOF >> "${MODULE_FILE}"
-			-- Normally we'd want to use Python from base Anaconda, but in this
-			-- special case we'll use Python from the environment itself.
+			-- Now using python from the environment is the default behavior
```

Comment:
I would just keep this as _"-- Using Python and tools from the environment"_

Reply:
(because generated modulefiles may have comments relevant to the module, but don't really need to mention anything about the generator tool behavioral changes)